site/src/Librecores/ProjectRepoBundle/Entity/ProjectClassification.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * A
+ *
+ * @ORM\Table(name="ProjectClassification")
+ * @ORM\Entity
+ */
+class ProjectClassification
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="categories", type="text")
+     */
+    private $categories;
```

Comment:
This is a string representing a single category/classification. Make this clear by using a singular term, e.g. "classification", or "category" (preference given to "classification", since that's equivalent to the name of this entity).